pymc/distributions/continuous.py

```diff
@@ -2271,6 +2297,17 @@ def logcdf(value, alpha, scale):
         )
         return check_parameters(res, 0 < alpha, 0 < beta, msg="alpha > 0, beta > 0")
 
+    def icdf(value, alpha, scale):
+        beta = pt.reciprocal(scale)
```

Comment:
If beta is just used for the assert, could instead assert scale and avoid the reciprocal

Reply:
Done! Even though checking for scale, I kept beta on the user facing message, because this is the parameterization used in the documentation. Fixed in https://github.com/pymc-devs/pymc/pull/6845/commits/be24e627a698659037db8cb8affb9c82ad24d2ee

Reply:
Sounds good